src/main/python/search_variants_example_brca1.py

```diff
@@ -0,0 +1,32 @@
+# export SPARK_CLASSPATH=...
+import json
+import pyspark
+import pyspark.rdd
+import pyspark.conf
+from pyspark.serializers import UTF8Deserializer
+
+conf = pyspark.conf.SparkConf()
+sc = pyspark.SparkContext(conf=conf)
+
+def brca1():
+    args = sc._gateway.new_array(sc._jvm.java.lang.String, 0)
+    jsc = sc._jsc.sc()
+
+    scala_rdd = (sc._jvm.com.google.cloud.genomics.spark.examples.
+        SearchVariantsExampleBRCA1.createJsonRdd(jsc, args))
```

Comment:
This can be done in a future PR but it would be cool to refactor what you are using from SearchVariantsExampleBRCA1 and not use SearchVariantsExampleBRCA1 directly.

The idea we want to convey with this example is that it shows how to re-write SearchVariantsExampleBRCA1 in Python - awesome!